group.py

```diff
@@ -19,25 +20,15 @@
 
 # Simple loop to extract ages from the group, and choose the largest
 
-max_age = 0
-
-for a in my_group:
-    age = my_group[a]['age']
-    if age > max_age:
-        max_age = age
+ages = [my_group[a]['age'] for a in my_group]
+max_age = max(ages)
 
 print("The oldest person in the group is",max_age,"years old.")
```

Comment:
also you can use f strings here, even calling functions inline

```suggestion
print(f"The oldest person in the group is {max(ages)} years old.")
```

Reply:
looks good thanks for the help